src/ducks/stats/services.js

```diff
@@ -0,0 +1,36 @@
+import moment from 'moment'
+import { BankTransaction } from 'cozy-doctypes'
+import { sumBy } from 'lodash'
+
+export const getPeriod = () => {
+  const end = moment()
+    .utc()
+    .subtract(1, 'month')
+    .endOf('month')
+
+  const start = end
+    .clone()
+    .subtract(2, 'month')
+    .startOf('month')
+
+  return { start, end }
+}
+
+export const fetchTransactionsForPeriod = period => {
+  return BankTransaction.queryAll({
+    date: {
+      $gte: period.start,
+      $lte: period.end
```

Comment:
I didn't see that before. You are doing "start of this month" -> "end of this month", both included.

If someone put fractions of seconds in the date, you may exclude a few documents. This is more a theorical case, but in the future you're better to do "start of this month" (included) -> "start of next month" (excluded)